### PR TITLE
Dare Mod

### PR DESCRIPTION
**Pull request by brennenhogan**

### CODE REVIEW

File: modules/tethics_dare.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+import json
+import random
+
+def dare_handler(sender, message, bot_id, app_id):
+    """Detain a user"""
+    message = message.strip().split()
+    if len(message) is 2:
+        dare = tehics_dares()
+        #dare = 'drink piss'
```
Comment (ginglis13) [inline modules/tethics_dare.py]: can this comment get yeeted out

File: modules/tethics_dare.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return "{} has been dared by {} to:\n{}\nTime to earn those participation points".format(message[1], sender, dare)
+    else:
+        return "Issue a tethics dare.\n usage: !dare <user>\n"
+
+
+def tehics_dares():
+	dares = []
+	#with open('/Users/brennen/School/Junior Year/shortstoploc/dares.txt') as f:
```
Comment (ginglis13) [inline modules/tethics_dare.py]: also this comment

**Review by ginglis13 — CHANGES_REQUESTED** [review]
looks good yo. i requested some changes. also, could you delete the .DS_Store file from this branch? I forgot to add those to the .gitignore, that's just a folder formatting file on macOS.

One other thing - could you update `app.py` to include this module?

File: modules/tethics_dare.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+import json
+import random
+
+def dare_handler(sender, message, bot_id, app_id):
+    """Detain a user"""
```
Comment (ginglis13) [inline modules/tethics_dare.py]: i guess this could be like "dare a user to do something during tethics"

**Review by ginglis13 — CHANGES_REQUESTED** [review]

**Comment by brennenhogan** [comment]
changes complete

**Review by ginglis13 — APPROVED** [review]
good stuff o7

**Pull request merged**